cypress/support/PageObject.js

```diff
@@ -2,6 +2,13 @@ class PageObject {
   visit(url) {
     cy.visit(url || this.url);
   }
+
+  confirmDelete() {
+    cy.on('window:confirm', (text) => {
+      expect(text).to.equal('Do you really want to delete it?');
```

Comment:
The confirmation text 'Do you really want to delete it?' should match exactly what is expected in the application. Ensure that this text is correct and update it if the application's confirmation message changes.